mongodbatlas/data_source_mongodbatlas_search_analyzers_test.go

```diff
@@ -0,0 +1,42 @@
+package mongodbatlas
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"os"
+	"testing"
+)
+
+func TestAccDataSourceMongoDBAtlasSearchAnalyzer_basic(t *testing.T) {
+	var (
+		clusterName = acctest.RandomWithPrefix("test-acc-global")
+		projectID   = os.Getenv("MONGODB_ATLAS_PROJECT_ID")
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckMongoDBAtlasSearchAnalyzerDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMongoDBAtlasSearchAnalyzerDSConfig(projectID, clusterName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Can we check here for other attributes to be set also?

Reply:
sure!